### DISCUSSION TITLE
`setup_ci` and `create_keychain` should set keychain name lane context variables

### DISCUSSION THREAD

**Discussion by alex-benoit** [fastlane/fastlane#16906] [Ideas] [answered]
Would be very handing for example when using `import_certificate`

**Comment by joshdholtz** [comment]
@alex-benoit I _really_ like this idea 😊 This would be a great add. I’d be open to a PR if you’d like to make it otherwise I’ll try add this onto my queue 💪

**Reply by alex-benoit** [reply]
Let me give it a shot! I'm new to contributing to fastlane, so might take a while!

**Comment by alex-benoit** [comment, marked as answer]
https://github.com/fastlane/fastlane/pull/16920
